demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -550,19 +547,16 @@ int main(int argc, char *argv[]) {
                     videoWriter.write(frame);
                 }
 
-                timer["total"].calculateDuration();
                 framesCounter++;
             }
-            timer.finish("total");
-
-            //// --------------------------- Report metrics -------------------------------------------------------
-            slog::info << "Metrics report:" << slog::endl;
-            slog::info << "\tNumber of processed frames: " << framesCounter << slog::endl;
-            slog::info << "\tFPS: " << framesCounter * (1000.0 / timer["total"].getTotalDuration()) << slog::endl;
-            slog::info << presenter->reportMeans() << slog::endl;
-        } while (FLAGS_loop);
+        } while (FLAGS_loop && !stop);
```

Comment:
I think, loop behavior have to aligned with other OMZ-GAPI demo with `ImagesCapture`. I try do this soon.